### ISSUE TITLE
More information in directory

### ISSUE THREAD

**Issue by dbrgn** [spacedirectory/directory#4]
Idea by @rorist:

> Finally, for the clients it'll be good to have more information in the directory. For instance when I'm showing the list of hackerspaces in MyHackerspace, I can only display the name of the hs, because that's all there is in the directory. It would be nice for the directory to fetch regularly information from the endpoints (maybe not the status, as it changes often) like the logo, country, location, etc.

**Comment by dns2utf8** [comment]
And maybe timestamps with when created, last successful fetch, ...?

**Comment by dbrgn** [comment]
Having such a solution with dynamic data would require a dynamic component though, so it's not "just" a plain static JSON file anymore. (Although a cronjob would probably enough.)

**Comment by rorist** [comment]
We'll probably need some scripts anyway, if we want to fulfill #2

**Comment by dbrgn** [comment]
Not really, since #2 would not be a function that every mirror should replicate. Otherwise people will get dozens of error notification mails :)
